### DISCUSSION TITLE
2024 fire update testing

### DISCUSSION THREAD

**Discussion by samsrabin** [NCAR/LMWG_dev#64] [General]
We need to do some scientific testing of the [2024 fire updates](https://github.com/orgs/ESCOMP/projects/44/views/1). This won't be possible until the fire surface dataset variables are updated, which hopefully should happen in the next couple of days. For now, let's use this Discussion to figure out what the tests should look like.

We could use either `li2024gswpfrc` or `li2024crujra`, but the latter is more likely to become the new default since 6.0 `I` cases will be using CRU-JRA.

We'll probably want to start from a spinup with the new fire, so I'll let @slevis-lmwg and @olyson decide when/how that should happen—again, contingent on having the new surface dataset variables ready.

@wwieder, do you want to add anything?

**Comment by slevis-lmwg** [comment]
In recent tests we have used the crujra data that ends in 2023. I suspect we will do the same unless we discover something wrong with the data.
